packages/lighthouse-spa/package.json

```diff
@@ -6,8 +6,8 @@
     "start": "npm run config -- --environment=dev && ng serve",
```

Comment:
Kindly bumb up the version number. Minor one

Reply:
okay will do that